tests/test_sbr_calculator.py

```diff
@@ -0,0 +1,353 @@
+from src.smart_building_rating_calculator.calculate_sbr_score import (
+    get_sbr_scores,
+    sbr_score,
+)
+from src.smart_building_rating_calculator.flex_archetype import FlexArchetype
+from src.smart_building_rating_calculator.inputs import (
+    BatterySize,
+    EVChargerPower,
+    HeatingSource,
+    HotWaterSource,
+    SolarInverterSize,
+    UserInputs,
+)
+from src.smart_building_rating_calculator.intermediate_scoring import (
+    calc_electrification_score,
+    calc_ics_score,
+    calc_smart_meter_score,
+)
+
+
+class TestScoreCalculators:
+    """Test the scoring calculations using the results in the SBR excel sheet created by ESC"""
+
+    def test_smartest_home(self):
```

Comment:
The proper way to do this is to parametrize the test by:

1. Creating these UserInputs as pytest fixtures
2. Call these fixtures and parametrize the test and compare it against expected results

https://engineeringfordatascience.com/posts/pytest_fixtures_with_parameterize/

Reply:
Nice, thanks, I have now implemented fixtures as arguments.